cloud/src/meta-service/meta_service_schema.cpp

```diff
@@ -119,4 +128,239 @@ bool parse_schema_value(const ValueBuf& buf, doris::TabletSchemaCloudPB* schema)
     return buf.to_pb(schema);
 }
 
+// Map item to dictionary key, and add key to rowset meta, if it is a new one, generate it and increase item id
+// Need to remove dynamic parts from original RowsetMeta's TabletSchema, to make fdb schema kv stable
+template<typename ItemPB>
+void process_dictionary(
+    SchemaCloudDictionary& dict,
+    const google::protobuf::Map<int32_t, ItemPB>& item_dict,
+    google::protobuf::RepeatedPtrField<ItemPB>* result,
+    RowsetMetaCloudPB* rowset_meta,
+    const google::protobuf::RepeatedPtrField<ItemPB>& items,
```

Comment:
suggestion for more readable: rename to columns_or_indexes or add comment

Reply:
comment added